dashboard/test/ui/step_definitions/applab.rb

```diff
@@ -182,6 +182,14 @@ def set_nth_input(n, value)
   @browser.execute_script(script)
 end
 
+And /^I drag the instructions grippy by ([-|\d]+) pixels$/ do |delta|
+  drag_grippy('$(".fa-ellipsis-h").eq(0).parent()', 0, delta)
```

Comment:
It might be worth adding an ID to the instructions grippy just to make our tests more resilient.  I guess we can _hope_ it will always be the first horizontal ellipsis...